PowerEditor/installer/nativeLang/estonian.xml

```diff
@@ -1,903 +1,1008 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-*Estonian language file for Notepad++*
-
-	** Created By:- Yusuf Mullaji *
-		Email Id-> yusufmullaji@rocketmail.com
-
+Estonian translation by Andres Traks
+https://github.com/AndresTraks/
```

Comment:
Thanks for the review!

The previous file was generated using an automatic translation process and practically none of the strings were right or made sense. It seems weird to include the info for such a translation.

Consider that tracking is already done perfectly in the repository since translation changes are required to be submitted as git pull requests.